CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+## v6.4.0 Next Release
```

Comment:
Nit: version numbers are picked out at release time based on what's included, this may not be the only thing changed, for example this is really a patch and not a feature so it will most likely be 6.3.1. Let's not pre-label versions in the future